spec/views/supervisors/index.html.erb_spec.rb

```diff
@@ -35,29 +35,29 @@
 
         parsed_html = Nokogiri.HTML5(rendered)
 
-        expect(parsed_html.css("#supervisors .supervisor_case_contact_stats .attempted-contact").length).to eq(1)
-        expect(parsed_html.css("#supervisors .supervisor_case_contact_stats .no-attempted-contact").length).to eq(1)
+        expect(parsed_html.css("#supervisors .success-bg").length).to eq(1)
+        expect(parsed_html.css("#supervisors .danger-bg").length).to eq(1)
       end
 
-      it "accurately displays the number of active and inactive volunteers per supervisor" do
-        create(:volunteer, :with_cases_and_contacts, supervisor: supervisor)
-        assign :supervisors, [supervisor]
-        render template: "supervisors/index"
-
-        parsed_html = Nokogiri.HTML5(rendered)
-
-        active_bar = parsed_html.css("#supervisors .supervisor_case_contact_stats .attempted-contact")
-        inactive_bar = parsed_html.css("#supervisors .supervisor_case_contact_stats .no-attempted-contact")
-        active_flex = active_bar.inner_html
-        inactive_flex = inactive_bar.inner_html
-        active_content = active_bar.children[0].text.strip
-        inactive_content = inactive_bar.children[0].text.strip
-
-        expect(active_flex).to eq(active_content)
-        expect(inactive_flex).to eq(inactive_content)
-        expect(active_flex.to_i).to eq(2)
-        expect(inactive_flex.to_i).to eq(1)
-      end
+      # it "accurately displays the number of active and inactive volunteers per supervisor" do
+      #   create(:volunteer, :with_cases_and_contacts, supervisor: supervisor)
+      #   assign :supervisors, [supervisor]
+      #   render template: "supervisors/index"
+
+      #   parsed_html = Nokogiri.HTML5(rendered)
+
+      #   active_bar = parsed_html.css("#supervisors .success-bg")
+      #   inactive_bar = parsed_html.css("#supervisors .danger-bg")
+      #   active_flex = active_bar.inner_html
+      #   inactive_flex = inactive_bar.inner_html
+      #   active_content = active_bar.children[0].text.strip
+      #   inactive_content = inactive_bar.children[0].text.strip
+
+      #   expect(active_flex).to eq(active_content)
+      #   expect(inactive_flex).to eq(inactive_content)
+      #   expect(active_flex.to_i).to eq(2)
+      #   expect(inactive_flex.to_i).to eq(1)
+      # end
```

Comment:
@rae-stanton is this test broken now? Can it be modified to match the new layout?

Reply:
@littleforest oops! I thought I committed the final test - it's passing now and I've added the updated commit for the test file.